qa/rpc-tests/wallet_orchard_change.py

```diff
@@ -65,17 +70,22 @@ def run_test(self):
         acct1 = self.nodes[1].z_getnewaccount()['account']
         ua1 = self.nodes[1].z_getaddressforaccount(acct1, ['orchard'])['address']
 
-        recipients = [{"address": ua1, "amount": 1}]
-        myopid = self.nodes[0].z_sendmany(ua0, recipients, 1, 0, 'AllowRevealedAmounts')
+        recipients = [{"address": ua1, "amount": Decimal('1')}]
+        # TODO The z_sendmany call fails when passed `null` because it calculates a fee
+        # that is too low. If we passed `fee` in here instead, it would succeed.
+        # https://github.com/zcash/zcash/issues/6956
+        fee = conventional_fee(4)
+        myopid = self.nodes[0].z_sendmany(ua0, recipients, 1, ZIP_317_FEE, 'AllowRevealedAmounts')
```

Comment:
What do we want to do here, pass `fee` and leave the bugfix for a point release? I think that's probably best.
```suggestion
        myopid = self.nodes[0].z_sendmany(ua0, recipients, 1, fee, 'AllowRevealedAmounts')
```

Reply:
Yes, I agree.